Sources/VaporToolbox/New/New.swift

```diff
@@ -6,62 +6,73 @@ struct New: Command {
     struct Signature: CommandSignature {
         @Argument(name: "name", help: "Name of project and folder.")
         var name: String
+        
+        @Option(name: "template", short: "T", help: "The URL of a Git repository to use as a template.")
```

Comment:
Why capitalised?

Reply:
I honestly couldn't tell you. For some reason it felt better that way. Still does 🤷‍♀ I'm not married to it.